### PR TITLE
[SD-4179] reorder the publish queue

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

File: superdesk/publish/publish_queue.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ class PublishQueueResource(Resource):
     }
 
     etag_ignore_fields = ['moved_to_legal']
-    datasource = {'default_sort': [('_created', -1)]}
+    datasource = {'default_sort': [('_created', -1), ('subscriber_id', 1), ('published_seq_num', -1)]}
```
Comment (akintolga) [inline superdesk/publish/publish_queue.py]: Does this line require a new index in Mongo?

**Comment by akintolga** [comment]
:+1:

File: superdesk/publish/publish_queue.py (hunk 1 above)
Comment (marwoodandrew) [inline superdesk/publish/publish_queue.py]: I would imagine it would.

Comment (marwoodandrew) [inline superdesk/publish/publish_queue.py]: I'd like to get this change merged and onto UAT then create the index manually, Once we're happy with it add the index creation to app_initialize.py

**Comment by mdhaman** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

File: features/publish_queue.feature [hunk 2]
```diff
@@ -153,4 +153,16 @@ Feature: Publish Queue
...
+  Scenario: publish queue us returned in correct order
+    Given "publish_queue"
+    """
+    [{"_id":4, "_created":"2016-05-30T10:00:00+00:00", "subscriber_id": 4, "published_seq_num": 2},
+    {"_id":3, "_created":"2016-05-30T10:00:00+00:00", "subscriber_id": 4, "published_seq_num": 3},
+    {"_id":2, "_created":"2016-05-30T10:00:00+00:00", "subscriber_id": 3, "published_seq_num": 1},
+    {"_id":1, "_created":"2016-05-30T10:00:00+00:00", "subscriber_id": 2, "published_seq_num": 1}]
+    """
+    When we get "/publish_queue"
+    Then we get list ordered by _id with 4 items
```
Comment (ahilles107) [inline features/publish_queue.feature]: missing empty new line at end of file

**Pull request merged**